templates/docs/dot-env.md

```diff
@@ -13,7 +15,7 @@ APP_LOG_LEVEL=debug
 APP_URL=https://myapp.com
 ```
 
-Buffalo ships with `.env` support (**since buffalo < 0.10.3**), meaning buffalo will load .env files into ENV variables once the application starts, to do it, buffalo uses [`Envy.Load`](https://github.com/gobuffalo/envy/blob/master/envy.go#L53) which will look for `.env` file at the root of your app.
+Buffalo ships with `.env` support (**since buffalo >= 0.10.3**), meaning buffalo will load .env files into ENV variables once the application starts, to do it, buffalo uses [`Envy.Load`](https://github.com/gobuffalo/envy/blob/e613c80275b86293880eddeb27417c9a7c670ff3/envy.go#L53) which will look for `.env` file at the root of your app.
 
-Generated apps (**with buffalo < 0.10.3**) will also create a default `.env` file in your application root, this file will be but will be listened by refresh for changes but will be ignored by git (added in the gitignore).
+Generated apps (**with buffalo >= 0.10.3**) will also create a default `.env` file in your application root, this file will be but will be listened by refresh for changes but will be ignored by git (added in the gitignore).
```

Comment:
Hum, I forgot a thing here. This sentence is a bit weird, isn't it?

Reply:
lol, yes!